.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+
+jobs:
+  build:
+    #working_directory: src/proxy
+    environment:
+    docker:
+      - image: costinm/proxy-builder:0.4.4
```

Comment:
You can copy from istio/istio. We have access to xlarge resources now.

Reply:
Not on my fork, where I've been testing. 
It's an interesting problem - we want people to be able to fork and use circleci on the fork. The xlrage will be ignored - but if I increase the parallelism (-j 16), the build will fail on the small
machine. Not sure how to customize the build, maybe some env settings ? 

Even with the small image - the build seems pretty fast, in particular for incremental builds.

Reply:
I don’t think we can optimize for forks here. It won’t work until they sign up for circle ci. Let’s optimjze for our build time. 15mins is a long time.